edudoc/delegate-handbook/delegate-handbook.md

```diff
@@ -512,6 +512,34 @@ Generation of groups is an organizer responsibility, but as a Delegate it is you
 
 For non-first rounds, it is a good practice to put all of the similarly skilled competitors in one group to ensure that they receive the same scrambles. Use the previous round’s results to determine which groups to place competitors.
 
+## Competition Cancellation and Rescheduling
+
+Once a competition is announced, it is the expectation and responsibility of the Delegates to commit to seeing the competition through on the original date announced. Competitors and their families will have made arrangements and financial commitments to attend on that day, and it is in everyone’s best interest to honor those commitments.
+
+A cancellation should only happen when there is no safe or feasible alternative. This is only an option in extenuating circumstances such as:
+
+- Extreme weather conditions/natural disasters
+- Political unrest or conflict
+- Venue facility failures (electrical, plumbing, structural), where another venue is not possible
+- General safety concerns with competitors traveling to or attending the competition
+- Force majeure
+
+Competitions may not be canceled for the following reasons:
+
+- Low registration numbers
+- Financial concerns
+- Delegate(s) suddenly unable to attend
+
+If you encounter issues such as these where canceling a competition appears necessary, do not make any major changes or statements to competitors until the WCAT and the WCA Board offer guidance. Here are examples of some preventative measures that can be taken:
+
+- If the listed delegate(s) can no longer attend due to an emergency, in some circumstances the WCA Board can appoint an experienced and trusted community member as a temporary delegate if no other delegates are available. In this case, take the following steps:
+
+1. Reply-all to the competition thread and add the WCA Board
+2. Explain why you are no longer able to attend and who you propose as a temporary delegate.
+3. The Board will vote on whether to appoint this person and respond with any concerns.
+
+- For financial concerns, such as low registration turnout, Delegates are responsible for reviewing the budget prior to announcement and to have a plan in the event the budget is not meeting expectations. It is strongly recommended that organizing is always done through ROs or a similar legal entity to protect individuals from liability, and to always have backup funds available. Competitions will not be canceled solely over a budget deficit.
```

Comment:
I'm a bit concerned about that last sentence. So what do you do if you DO end up with a budget deficit? Are we expecting the Delegates to eat the cost themselves, or even worse, take out debt?

Reply:
WCA Delegates do not organise competitions - organisers do. In the event there is a loss on a competition, then the organiser would be responsible for that loss. That is why it is recommended that ROs or similar legal entities established for the purpose of organising competitions do so - they can absorb losses on some competitions and offset that with profits on others.

Reply:
Adding that it is the Board's position that competitions will not be cancelled solely over a budget deficit, as such things should never occur and are part of a Delegate's responsibility in reviewing financial viability.

@EdHollingdale would you be ok with a statement such as the following?

> Contact the WCA Financial Committee at the soonest indication that a budget deficit poses a significant risk to a competition.

Reply:
Seems like good wording - they can consult with WFC and we can try to support, and if it then escalates to actual consideration of cancellation then the Board/WCAT can be brought into the conversation as required.